lib/tests/cleanup_before.rb

```diff
@@ -28,9 +28,8 @@ def run!(args:)
           elsif OS.mac?
             if Dir.glob("#{HOMEBREW_CELLAR}/*").present?
               # moving is much faster than deleting.
-              tmp_dir = "/tmp/test-bot-cleanup"
-              test "bash", "-c", "mkdir -p #{tmp_dir}"
-              test "bash", "-c", "mv #{HOMEBREW_CELLAR}/* #{tmp_dir}"
+              test "bash", "-c", "mkdir -p /tmp/test-bot-cleanup"
```

Comment:
```suggestion
              FileUtils.mkdir_p "/tmp/test-bot-cleanup"
```

Untested but not reason to shell out here.

Reply:
I think that should work